test/peer_test.go

```diff
@@ -2,6 +2,7 @@ package test
 
```

Comment:
this is not the current version of tests on the main branch. Mabye need to pull?  In theory there shouldn't be any diffs here.

Reply:
now up do date